test/servinge2e/servicemesh_test.go

```diff
@@ -0,0 +1,123 @@
+package servinge2e
+
+import (
+	"github.com/openshift-knative/serverless-operator/test"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgTest "knative.dev/pkg/test"
+	"testing"
+)
+
+const (
+	// A test namespace that is part of the ServiceMesh (setup by "make install-mesh")
+	serviceMeshTestNamespaceName = "default"
+	serviceMeshTestImage = "gcr.io/knative-samples/helloworld-go"
+)
+
```

Comment:
Maybe it would be good to add a comment here that this test will only run if Service Mesh was installed. It's not clear until the middle of the test where it's optionally skipped.
The rest of this PR looks good to me.

Reply:
added 

> Skipped unless ServiceMesh has been installed via "make install-mesh"